### ISSUE TITLE
Layout issue when using drawer with react-native-scrollable-tab-view

### ISSUE THREAD

**Issue by Kureev** [root-two/react-native-drawer#127]
Hi there,

I'm using react-native-drawer with react-native-scrollable-tab-view. I want to achieve a material design drawer functionality, but seems something goes wrong when I'm using `type: 'overlay'`.

Basically, I don't see a content of my `ScrollableTabView` children, but header (see `renderTabBar`) works fine. After some investigations I ended up at the drawer default style. Seems that the issue happens because of `position: absolute` on the drawer component. Removing this rule breaks up visual appearance (because of `openDrawerOffset` it looks a bit opened), but the underlying layers works fine.

cc @brentvatne

**Comment by rt2zz** [comment]
I will try to take a look at this next week, the drawer definitely takes some liberties in laying out things within its provided view, and it is possible there could be conflicts with other components that try to similarly handle layout.

**Comment by rt2zz** [comment]
I made a little example app for scrollable tab view and it appears to be working: https://github.com/root-two/react-native-drawer/blob/master/examples/RNDrawerExample/src/Main.ScrollableTabView.js

There may be something specific about your set up. Can you share a gist with your drawer & tab view code?

**Comment by Kureev** [comment]
It seems that in the example [you're using `type="static"`](https://github.com/root-two/react-native-drawer/blob/master/examples/RNDrawerExample/src/App.js#L29) which works for me as well. I encounter an issue only when I use `type="overlay"`

**Comment by rt2zz** [comment]
ah, my component structure is a little wonky, the wrapper for that test is using overlay: https://github.com/root-two/react-native-drawer/blob/master/examples/RNDrawerExample/src/App.ScrollableTabView.js#L29

**Comment by rt2zz** [comment]
Is your drawer a parent of the scrollable tabview or vice versa?

**Comment by Kureev** [comment]
Drawer is a parent

**Comment by tlvenn** [comment]
@rt2zz have you been able to reproduce the issue ?

**Comment by rt2zz** [comment]
@tlvenn no, are you seeing a similar issue? if so please post the details
